acceptance-tests/tests/commands/config.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { describe, beforeAll, it, expect, afterAll } from 'vitest';
+import { TestState } from '../../lib/TestState';
+import { DOWN, ENTER } from '../../lib/prompt';
+
+describe('hs config', () => {
+  let testState: TestState;
+
+  beforeAll(async () => {
+    testState = new TestState();
+    await testState.withAuth();
+  });
+
+  afterAll(() => {
+    testState.cleanup();
+  });
+
+  describe('hs config set', () => {
+    it('should set the default mode to draft', async () => {
+      await testState.cli.executeWithTestConfig(
+        ['config', 'set'],
+        [ENTER, DOWN, ENTER]
+      );
```

Comment:
Might be misinterpreting this, but is this actually checking anything? Or is the check just that this doesn't fail?

Reply:
Ah good catch. I had actually forgotten to finish this one. I just added an actual check in here in my latest commit